src/mame/act/victor9k.cpp

```diff
@@ -179,8 +184,21 @@ class victor9k_state : public driver_device
 //  MACROS / CONSTANTS
 //**************************************************************************
 
-#define LOG 0
+#define LOG_CONF      (1 << 1U)
+#define LOG_KEYBOARD  (1 << 2U)
+#define LOG_DISPLAY   (1 << 3U)
+
+//#define VERBOSE (LOG_CONF|LOG_DISPLAY|LOG_KEYBOARD)
+//#define LOG_OUTPUT_STREAM std::cout
+
+#define VERBOSE (LOG_CONF|LOG_DISPLAY)
+#define LOG_OUTPUT_STREAM std::cout
 
+#include "logmacro.h"
+
+#define LOGCONF(...)     LOGMASKED(LOG_CONF,  __VA_ARGS__)
+#define LOGKEYBOARD(...) LOGMASKED(LOG_KEYBOARD,  __VA_ARGS__)
+#define LOGDISPLAY(...)  LOGMASKED(LOG_DISPLAY,  __VA_ARGS__)
```

Comment:
This should go after all the other includes at the top of the file.

Reply:
Moved this up.